docs/getting-started.md

```diff
@@ -39,7 +39,7 @@ When installing these dependencies keep in mind that they should try to match th
 1.  Check if the project is running with `valet links`. You should be able to
     see your project; `https://<name-of-project>.test`
 1.  Run `mysql` to log into the locally running mariadb database.
-    - Inside the MySQL prompt create a new database by calling `CREATE DATABASE `<name-of-project>` CHARACTER SET UTF8mb4 COLLATE utf8mb4_danish_ci;`. Trivia: We use `utf8mb4_danish_ci` to ensure proper ordering of ÆØÅ.
+    - Inside the MySQL prompt create a new database by calling ``CREATE DATABASE `<name-of-project>` CHARACTER SET UTF8mb4 COLLATE utf8mb4_danish_ci;``. Trivia: We use `utf8mb4_danish_ci` to ensure proper ordering of ÆØÅ.
```

Comment:
```suggestion
    - Inside the MySQL prompt create a new database by calling `CREATE DATABASE <name-of-project> CHARACTER SET UTF8mb4 COLLATE utf8mb4_danish_ci;`. Trivia: We use `utf8mb4_danish_ci` to ensure proper ordering of ÆØÅ.
```

Wouldn't this do the same thing? 💭

Reply:
Nope, because the name of the project actually needs those backticks surrounding it or it wont be a valid query. I tried using `"` and `'` instead, and get problems with the query like so :
```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''keepit2' CHARACTER SET UTF8mb4 COLLATE utf8mb4_danish_ci' at line 1
``` 
Therefore we need to include the backticks in the example-command. 
In order to include the backticks, we need to use the `special` way of escaping them that I link to.
If you have tried this and have a different experience with manually creating databases with a mysql-command, I would like to know. 

Jeg tror også jeg brukte engelsk siden jeg skrev commits på engelsk. 😅

Reply:
Ah, but you can just skip any backticks 🙃 , just tried that, that would be equivalent.

Reply:
This change was originally made in response to this internal error-report: https://netlifedesign.slack.com/archives/C0NVCE55K/p1664885762194089?thread_ts=1664878387.350559&cid=C0NVCE55K

Reply:
Ie, this change solves this problem: https://stackoverflow.com/questions/35592034/how-do-i-add-database-name-with-hyphen-character-using-script-in-ubuntu

Reply:
An alternative change would be to remove the hyphens from the example-command and link to the valid characters that are allowed in the database-names.

Reply:
Link to specification of allowed characters: https://dev.mysql.com/doc/refman/5.7/en/identifiers.html#:~:text=Permitted%20characters%20in%20unquoted%20identifiers

Reply:
After this feedback and some research I am now of the belief that the actual problem here is the existence of hyphens in the example-command. This implies to new users that hyphens are allowable characters and might dead-end them when they attempt to run the suggested command with their database-name instead. 🤔

Reply:
This needs to be fixed since the current merged solution is wrong, it skips over the variable/project-name


![image](https://user-images.githubusercontent.com/5453010/199549204-c790f375-0549-4c41-9030-53dcf111730c.png)